src/controller/python/OpCredsBinding.cpp

```diff
@@ -465,6 +465,21 @@ PyChipError pychip_OpCreds_AllocateController(OpCredsContext * context, chip::Co
     return ToPyChipError(CHIP_NO_ERROR);
 }
 
+PyChipError pychip_OpCreds_InitGroupTestingData(chip::Controller::DeviceCommissioner * devCtrl)
```

Comment:
A better approach would be to make available APIs that give access to the actual GroupDataProvider keys and group mapping from Python. Otherwise this "testing data" is the only one that can ever work at all...

Reply:
Agreed that this only works with testing data. I figured that that could be added later in a follow up, this paves the way just to get there and offers insight into how one could approach that

Reply:
Opened a github issue to track adding these APIs later https://github.com/project-chip/connectedhomeip/issues/25247